config/nightwatch.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
   test_workers: false,
   test_settings: {
     'default': {
+      globals: {
+        buildtype: process.env.BUILDTYPE
```

Comment:
@jkassemi  Can you look at this change to our nightwatch config and comment? Alternatively, I can define a global to guard this feature, but in that case, where would be the best place to set that global so that it is available to both our e2e tests and to https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/js/edu-benefits/routes.jsx (and so that it enables the app in development but not in production)?